openshift_xinetd_example.conf

```diff
@@ -0,0 +1,49 @@
+# https://linux.die.net/man/5/xinetd.conf
+
+service openshift-api
+{
+    flags           = IPv4
+    bind            = <Lab_Host_IP>
+    disable         = no
+    type            = UNLISTED
+    socket_type     = stream
+    protocol        = tcp
+    user            = root
+    wait            = no
+    redirect        = <IPv6_API_Address> 6443
+    port            = 6443
+    only_from       = 0.0.0.0/0
+    per_source      = UNLIMITED
+}
+
+service openshift-ingress-ssl
+{
+    flags           = IPv4
+    bind            = <Lab_Host_IP>
+    disable         = no
+    type            = UNLISTED
+    socket_type     = stream
+    protocol        = tcp
+    user            = root
+    wait            = no
+    redirect        = <IPv6_Ingress_Address> 443
+    port            = 443
+    only_from       = 0.0.0.0/0
+    per_source      = UNLIMITED
+}
+
+service openshift-ingress
+{
+    flags           = IPv4
+    bind            = <Lab_Host_IP>
+    disable         = no
+    type            = UNLISTED
+    socket_type     = stream
+    protocol        = tcp
+    user            = root
+    wait            = no
+    redirect        = <IPv6_Ingress_Address> 80
+    port            = 80
```

Comment:
Numerous people told me that sshutle doesn't work with IPv6.
Although its documentation seems to state otherwise. What's the verdict?

Reply:
I've switched to port 8080 and after reinstalling dev-scripts everything seems to be working correctly.
Although the `openshift-ingress` service might be redundant.

Reply:
> Numerous people told me that sshutle doesn't work with IPv6.
> Although its documentation seems to state otherwise. What's the verdict?

sshuttle supports multiple backends via the `--method` arg - whatever it uses by default (which is `auto` but I'm not sure what implementation that translates to in our environments) doesn't work for ipv6, but according to the tproxy docs that method does, but it looks like it requires some additional host configuration.